modules/ROOT/pages/configure-client-management-openid-task.adoc

```diff
@@ -26,20 +26,18 @@ The following table contains examples of the URLs you need to supply, depending
 | Token Introspection | {BASE URL}/introspect | {BASE URL}/introspect | {BASE URL}/introspect
 |===
 
-. Sign into the master organization in Anypoint Platform with the *Organization Administrator* role.
-. In Anypoint Platform, click *Access Management*. 
-. In the menu on the left, click *External Identity*.
-. In the drop-down, select *Client Management > OpenID Connect Dynamic Client Registration*.
+. Sign in to the master organization in Anypoint Platform with the organization administrator role.
```

Comment:
nit: I know this was like this before, but you don't sign in to (master) organizations. You sign in to your user account and navigate to the master organization view in Access Management

Reply:
```suggestion
. Sign in to Anypoint Platform using an account that has the organization administrator role.
```